imagecraft/application.py

```diff
@@ -33,6 +38,16 @@
 class Imagecraft(Application):
     """Imagecraft application definition."""
 
+    def __init__(
+        self,
+        app: craft_application.AppMetadata,
+        services: service_factory.ServiceFactory,
+    ) -> None:
+        super().__init__(app=app, services=services, extra_loggers={"imagecraft"})
```

Comment:
The preferred way to do this is to pass `extra_loggers={"imagecraft"}` when instantiating the app in `cli._create_app`.

Reply:
Thanks! I think I applied what I found in charmcraft without properly checking if there was a better way.